laceworksdk/api/container_registries.py

```diff
@@ -0,0 +1,210 @@
+
+    def get(self,
+            guid=None,
+            type=None,
+            org=False):
+        """
+        A method to get all container registries.
+
+        :param guid: A string representing the container registry GUID.
+        :param type: A string representing the container registry type.
+        :param org: A boolean representing whether the request should be performed
+            at the Organization level
+
+        :return response json
+        """
+
+        logger.info("Getting container registry info from Lacework...")
+
+        # Build the Container Registries request URI
+        if guid:
+            api_uri = f"/api/v2/ContainerRegistries/{guid}"
+        elif type:
+            api_uri = f"/api/v2/ContainerRegistries/{type}"
+        else:
+            api_uri = "/api/v2/ContainerRegistries"
+
+        response = self._session.get(api_uri, org=org)
+
+        return response.json()
+
+    def get_by_type(self,
+                    type,
+                    org=False):
+        """
+        A method to get all container registries by type.
+
+        :param type: A string representing the container registry type.
+        :param org: A boolean representing whether the request should be performed
+            at the Organization level
+
+        :return response json
+        """
+
+        return self.get(type=type, org=org)
+
+    def get_by_guid(self,
+                    guid,
+                    org=False):
+        """
+        A method to get all container registries.
+
+        :param guid: A string representing the container registry GUID.
+        :param org: A boolean representing whether the request should be performed
+            at the Organization level
+
+        :return response json
+        """
+
+        return self.get(guid=guid, org=org)
+
+    def search(self,
+               query_data=None,
+               org=False):
+        """
+        A method to search container registries.
+
+        :param query_data: A dictionary containing the desired search parameters.
+            (filters, returns)
+
+        :return response json
+        """
+
+        logger.info("Searching container registries from Lacework...")
+
+        # Build the Container Registries request URI
+        api_uri = "/api/v2/ContainerRegistries/search"
+
+        response = self._session.post(api_uri, data=query_data, org=org)
+
+        return response.json()
+
+    def update(self,
+               guid,
+               name=None,
+               type=None,
+               enabled=None,
+               data=None,
+               org=False):
+        """
+        A method to update an container registry.
+
+        :param guid: A string representing the container registry GUID.
+        :param name: A string representing the container registry name.
+        :param type: A string representing the container registry type.
+        :param enabled: An integer representing whether the container registry is enabled.
```

Comment:
nit. would it be better to use a python boolean here and just translate for data if it needs an int?